tests/collections/asr/test_diar_utils.py

```diff
@@ -35,130 +47,289 @@ def check_range_values(target, source):
     return all(bool_list)
 
 
-def matrix(mat, torch=True):
-    if torch:
-        return torch.tensor(mat)
+def check_labels(target, source):
+    bool_list = []
+    for x, y in zip(target, source):
+        bool_list.append(abs(x - y) < 1e-6)
+    return all(bool_list)
+
+
+def matrix(mat, use_tensor=True, dtype=torch.long):
+    if use_tensor:
+        mat = torch.Tensor(mat).to(dtype)
     else:
-        return np.array(mat)
+        mat = np.array(mat)
+    return mat
 
 
-def generate_mock_emb(n_emb_per_spk, perturb_sigma, emb_dim):
-    """Generate a set of artificial embedding vectors from random numbers
+def generate_orthogonal_embs(total_spks, perturb_sigma, emb_dim):
+    """Generate a set of artificial orthogonal embedding vectors from random numbers
     """
-    return torch.rand(1, emb_dim).repeat(n_emb_per_spk, 1) + perturb_sigma * torch.rand(n_emb_per_spk, emb_dim)
+    gaus = torch.randn(emb_dim, emb_dim)
+    _svd = torch.linalg.svd(gaus)
+    orth = _svd[0] @ _svd[2]
```

Comment:
Isn't `@` for matrix multiplication removed from python with 3.9?

Reply:
As far as I know, `a @ b` is equivalent to `np.matmul(a, b)` and introduced in python 3.5 (quite recently)
Checkout python.org documents: 
https://docs.python.org/3/whatsnew/3.5.html#whatsnew-pep-465

It works with no problem with python 3.9,
but correct me if I am wrong, if it is deprecated.
I don't see any document saying that `@` is deprecated.